.github/workflows/e2e-dispatch.yml

```diff
@@ -50,6 +50,8 @@ jobs:
     # template: .action_templates/steps/checkout.yaml
```

Comment:
the submodule changes in this file should instead be added to https://github.com/mongodb/mongodb-kubernetes-operator/blob/fb205a37567b9cc8edc89503bc8f3ddcba249317/.action_templates/steps/checkout.yaml#L1 and https://github.com/mongodb/mongodb-kubernetes-operator/blob/fb205a37567b9cc8edc89503bc8f3ddcba249317/.action_templates/steps/checkout-fork.yaml#L1-L0 instead.

These are the templates that are used to generate the files under `.github` in the pre-commit hook